docs/runbooks/VirtOperatorRESTErrorsBurst.md

```diff
@@ -1,25 +1,62 @@
+<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+
 # VirtOperatorRESTErrorsBurst 
 
 ## Meaning
 
-More than 80% of the rest calls in virt-operator pods failed in the last 5 minutes. 
+This alert fires when more than 80% of the REST calls in the `virt-operator` pods failed in the last 5 minutes. This usually indicates that the `virt-operator` pods cannot connect to the API server. 
+
+This error is frequently caused by one of the following problems:
+
+- The API server is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the API server, and view its response times and overall calls.
+
+- The `virt-operator` pod cannot reach the API server. This is commonly caused by DNS issues on the node and networking connectivity issues.
 
 ## Impact
 
-This alert suggests that the virt-operator is likely lost the connection to the apiserver. Cluster-level actions such as upgrading and controller reconciliation will likely not function. Customer workloads, i.e. VM and VMI, should not be affected.
+Cluster-level actions, such as upgrading and controller reconciliation, probably do not function. 
+
+However, customer workloads, such as virtual machines (VMs) and VM instances (VMIs), are not likely to be affected.
 
 ## Diagnosis
 
-Two common types of errors may lead to this alert:
-- The apiserver is overloaded and we run into timeouts. Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls. For users without cluster privileges, logs of KubeVirt apiserver pods can be fetched with `kubectl logs` command.
-- The virt-operator pod cannot reach the apiserver. Common issues are network connectivity issues such as DNS issues on the node. Check virt-operator logs to verify whether it can connect to the apiserver at all.
-    - `export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"`
-    - `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-operator`
-    - `kubectl -n $NAMESPACE logs <pod-name>`.
-    - `kubectl -n $NAMESPACE describe pod <pod-name>`.
+Check whether `virt-operator` can connect to the API server.
```

Comment:
I think we can remove this line because the procedure does more than check connectivity issues. The whole procedure is diagnostic, so we do not need this sentence.